### PR TITLE
fix: [CCM-9239]: Always Show Test Connection Button

### PR DESCRIPTION
**Pull request by rajarshee-harness**
### Summary

This PR adds Test Connection button for all the connectors on cloud integration's page (previously it was present only for connectors with error status). It also adds Delete Connector button in Cloud Accounts tab.

#### Screenshots

Screenshot 2022-09-29 at 12 23 59 PM

Screenshot 2022-09-29 at 12 24 08 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11487)

**Review by jyotiarora-harness — APPROVED** [review]

File: src/modules/75-ce/components/CloudIntegrationTabs/CloudAccountsTab.tsx [hunk 1]
```diff
@@ -165,13 +175,43 @@ const MenuCell: CustomCloudCell = ({ row, column }) => {
...
-  const [canUpdate] = usePermission(
+  const { mutate: deleteConnector } = useDeleteConnector({
+    queryParams: {
+      accountIdentifier: accountId
+    }
+  })
+
+  const { openDialog } = useConfirmationDialog({
+    titleText: getString('connectors.confirmDeleteTitle'),
+    contentText: `${getString('connectors.confirmDelete')} ${connector?.name}?`,
```
Comment (jenil-harness) [inline src/modules/75-ce/components/CloudIntegrationTabs/CloudAccountsTab.tsx:186]: knit: Define string as `delete text {{ name }}`

File: src/modules/75-ce/components/CloudIntegrationTabs/K8sClustersTab.tsx [hunk 2]
```diff
@@ -245,18 +245,16 @@ const FeaturesEnabledCell: CustomK8sCell = ({ row, column }) => {
...
-            />
-          ) : null}
+          <Button
+            icon="repeat"
+            iconProps={{ size: 12 }}
+            variation={ButtonVariation.LINK}
+            text={getString('common.smtp.testConnection')}
+            size={ButtonSize.SMALL}
+            onClick={() => {
+              ;(column as any).openTestConnectionModal({ connector: ccmk8sConnector })
```
Comment (jenil-harness) [inline src/modules/75-ce/components/CloudIntegrationTabs/K8sClustersTab.tsx:255]: Not getting type for column?

**Review by jenil-harness — APPROVED** [review]

File: src/modules/75-ce/components/CloudIntegrationTabs/K8sClustersTab.tsx (hunk 2 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/components/CloudIntegrationTabs/K8sClustersTab.tsx:255]: Screenshot 2022-09-30 at 2 42 01 PM

Yes, I don't see the type if I pass it like this to a cell.

**Pull request merged**